src/components/PersonDetails/index.tsx

```diff
@@ -82,6 +85,43 @@ const PersonDetails: React.FC = () => {
     return <Error statusCode={404} />;
   }
 
+  const personAttributes: string[] = [];
+
+  if (data.birthday) {
+    if (data.deathday) {
+      personAttributes.push(
+        intl.formatMessage(messages.lifespan, {
+          birthdate: intl.formatDate(data.birthday, {
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+          }),
+          deathdate: data.deathday
+            ? intl.formatDate(data.deathday, {
+                year: 'numeric',
+                month: 'long',
+                day: 'numeric',
+              })
+            : null,
```

Comment:
You don't need to check if `data.deathday` is set here.

Reply:
Oh yup, that was leftover from before I changed the string. Will fix shortly!